pegasus/cache/i18n/en-US.yml

```diff
@@ -375,6 +375,7 @@
   homepage_slot_text_blurb_educators: "Teach your students"
   homepage_slot_text_link_elementary: "Elementary school"
   homepage_slot_text_link_middle: "Middle school"
+  homepage_slot_text_link_middle_and_high: "Middle and High School"
```

Comment:
This needs to go in the [i18n gsheet](https://docs.google.com/spreadsheets/d/1Tq7VqZALgRA0wYk0HDfEOTyRI0TM2Dir2rloXIPGCgU/edit#gid=0&range=342:343).